content/docs/gherkin/reference.md

```diff
@@ -442,22 +442,29 @@ In order to allow Gherkin to be written in a number of languages, the keywords h
 
 ### List translation options
 You can get information about the translations from the command line.
+Note: 'cucumber' command may be different over code languages. ex. for Node.js is 'cucumber-js'
 
 To see a listing of available languages:
 ```shell
-cucumber --i18n help
+cucumber --i18n-languages help
```

Comment:
Unfortunately, this option is named differently across platforms, so changing from `--i18n` to `--i18n-languages` makes this incorrect for cucumber-ruby and cucumber-jvm.

I think we should simply get rid of the option and refer users to the docs.

Reply:
See https://github.com/cucumber/cucumber/issues/783